multimodal/src/autogluon/multimodal/models/ft_transformer.py

```diff
@@ -1,8 +1,13 @@
+import os
+import tempfile
 from typing import List, Optional
 
 import torch
 from torch import Tensor, nn
 
+from autogluon.common.loaders._utils import download
+from autogluon.common.utils.s3_utils import is_s3_url
```

Comment:
Can we use these utils?
https://github.com/autogluon/autogluon/blob/master/multimodal/src/autogluon/multimodal/utils/download.py#L31
https://github.com/autogluon/autogluon/blob/master/multimodal/src/autogluon/multimodal/utils/download.py#L49